app-ios/Modules/Sources/Theme/Typography.swift

```diff
@@ -25,19 +25,19 @@ public struct TextStyle {
 }
 
 public struct TypographyTokens {
-    public static let displayLarge = TextStyle(font: SwiftUI.Font.custom("Montserrat-Regular", size: 57), lineHeight: 64)
-    public static let displayMedium = TextStyle(font: SwiftUI.Font.custom("Montserrat-Regular", size: 45), lineHeight: 52)
-    public static let displaySmall = TextStyle(font: SwiftUI.Font.custom("Montserrat-Regular", size: 36), lineHeight: 44)
-    public static let headlineLarge = TextStyle(font: SwiftUI.Font.custom("Montserrat-Medium", size: 32), lineHeight: 40)
-    public static let headlineMedium = TextStyle(font: SwiftUI.Font.custom("Montserrat-Medium", size: 28), lineHeight: 36)
-    public static let headlineSmall = TextStyle(font: SwiftUI.Font.custom("Montserrat-Medium", size: 24), lineHeight: 32)
-    public static let titleLarge = TextStyle(font: SwiftUI.Font.custom("Montserrat-Regular", size: 22), lineHeight: 28)
-    public static let titleMedium = TextStyle(font: SwiftUI.Font.custom("Montserrat-SemiBold", size: 16), lineHeight: 24, letterSpacing: 0.15)
-    public static let titleSmall = TextStyle(font: SwiftUI.Font.custom("Montserrat-SemiBold", size: 14), lineHeight: 20, letterSpacing: 0.1)
-    public static let labelLarge = TextStyle(font: SwiftUI.Font.custom("Montserrat-SemiBold", size: 14), lineHeight: 20, letterSpacing: 0.1)
-    public static let labelMedium = TextStyle(font: SwiftUI.Font.custom("Montserrat-SemiBold", size: 12), lineHeight: 16, letterSpacing: 0.5)
-    public static let labelSmall = TextStyle(font: SwiftUI.Font.custom("Montserrat-SemiBold", size: 11), lineHeight: 16, letterSpacing: 0.5)
-    public static let bodyLarge = TextStyle(font: SwiftUI.Font.custom("Montserrat-Regular", size: 16), lineHeight: 24, letterSpacing: 0.15)
-    public static let bodyMedium = TextStyle(font: SwiftUI.Font.custom("Montserrat-Regular", size: 14), lineHeight: 20, letterSpacing: 0.25)
-    public static let bodySmall = TextStyle(font: SwiftUI.Font.custom("Montserrat-Regular", size: 12), lineHeight: 16, letterSpacing: 0.4)
+    public static let displayLarge = TextStyle(font: SwiftUI.Font.custom("Montserrat-Regular", size: 57), lineHeight: 64 - 57)
```

Comment:
Sorry, I missed this review comment 🙏🏼 
Can you apply this for all vairables?

https://github.com/DroidKaigi/conference-app-2023/pull/1135#discussion_r1320616538

Reply:
@ry-itto 
Sorry if I am wrong in my perception. :bow: 
I only commented on one area, but I have already corrected all the variables this time.

Reply:
@Corvus400 
My review comment is bad.. (That review comment has two meanings..)
What I want to say is "using static access to font name might be better way". 🙏🏼 

Like this
`FontAssets.Montserrat.regular.font(size: 57`

Reply:
@ry-itto 
Is [this](https://github.com/DroidKaigi/conference-app-2023/pull/1178/commits/e9e1c83a7d70e4509a67dcab4f2e5b2d16454d7b) correction correct?